modules/test/playwright/tests/object-web/objectDefinitions.spec.ts

```diff
@@ -88,4 +88,150 @@ test.describe('Manage object definitions through Model Builder', () => {
 			objectDefinition.id
 		);
 	});
+
+	test('can delete an published object definition by model builder', async ({
+		apiHelpers,
+		modalAddObjectDefinitionPage,
+		modelBuilderPage,
+	}) => {
+		const objectDefinition1 =
+			await apiHelpers.objectAdmin.postRandomObjectDefinition({
+				objectFolderExternalReferenceCode: 'default',
+				status: {code: 0},
+			});
+
+		const objectDefinition2Label = 'ObjectDefinition' + getRandomInt();
+
+		await modelBuilderPage.goto({objectFolderName: 'Default'});
+
+		await modelBuilderPage.createNewObjectDefinitionButton.click();
+
+		const objectDefinition2 =
+			await modalAddObjectDefinitionPage.createObjectDefinition(
+				objectDefinition2Label
+			);
+
+		await modelBuilderPage.clickToggleSidebarsButton();
+
+		await modelBuilderPage.clickFitViewButton();
+
+		await modelBuilderPage.clickObjectDefinitionActionsButton(
+			objectDefinition1.name
+		);
+
+		await modelBuilderPage.deleteObjectDefinition(objectDefinition1.name);
+
+		await expect(
+			modelBuilderPage.objectDefinitionNodes.filter({
+				hasText: objectDefinition2.name,
+			})
+		).toBeVisible();
+
+		await expect(
+			modelBuilderPage.objectDefinitionNodes.filter({
+				hasText: objectDefinition1.name,
```

Comment:
can you please pass the label instead of the name?

Reply:
yes, sure! I've done it to all of them 😄